packages/clerk-js/src/core/clerk.ts

```diff
@@ -908,10 +914,31 @@ export default class Clerk implements ClerkInterface {
     this.#componentControls?.updateProps(props);
   };
 
+  #syncWithPrimary = () => {
+    const q = new URLSearchParams({
+      redirect_url: `${window.location.href}?synced=true`,
+    });
+    window.location.replace(new URL(`/v1/client/sync?${q.toString()}`, `https://${this.domain}`).toString());
+  };
+
   #loadInStandardBrowser = async (): Promise<void> => {
+    console.log('----- isSatellite -----', this.isSatellite);
+    if (this.isSatellite) {
+      const q = new URLSearchParams(window.location.search);
+      if (!q.has('synced')) {
+        // Navigate to main FAPI and try to sync cookies
```

Comment:
:upside_down_face: Let's remove this comment.

Reply:
👍